tests/symbolicator/__init__.py

```diff
@@ -26,10 +26,20 @@ def strip_frame(frame):
     return frame
 
 
+NORMALIZED_REGISTERS = {"eflags": "efl"}
+
+
+def normalize_register(name):
+    return NORMALIZED_REGISTERS.get(name) or name
+
+
 def strip_stacktrace(stacktrace):
     if stacktrace:
         stacktrace = dict(stacktrace)
         stacktrace["frames"] = [strip_frame(x) for x in stacktrace.get("frames") or ()]
+        stacktrace["registers"] = {
+            normalize_register(k): v for k, v in (stacktrace.get("registers") or {}).items()
+        }
```

Comment:
```suggestion
        try:
            stacktrace["registers"] = {
                normalize_register(k): v for k, v in stacktrace["registers"].items()
            }
        except KeyError:
            pass
```
This is not only easier to read but also will not create an empty dictionary and add it to the snapshots